src/arch/helperadvsimd.h

```diff
@@ -130,11 +130,11 @@ static INLINE vfloat vsqrt_vf_vf(vfloat d) { return vsqrtq_f32(d); }
 
 // Multiply accumulate: z = z + x * y
 static INLINE vfloat vmla_vf_vf_vf_vf(vfloat x, vfloat y, vfloat z) {
-  return vmlaq_f32(z, x, y);
+  return vfmaq_f32(z, x, y);
```

Comment:
This LGTM, I was just asking you to provide an example of code that generated multiplication and addition (separated) from the `vmlaq_f32` intrinsics.

No worries if you don't have time.